custom_components/tuya_local/devices/tuya_gas_sensor.yaml

```diff
@@ -0,0 +1,12 @@
+name: TUYA Gas Leak Detector
+products:
+  - name: TUYA Gas Leak Detector
```

Comment:
Products list must have id for each product, otherwise it will not be able to detect which is which.
Branding in name should be actual brand and model, I think Tuya is not selling any end products themselves.

Reply:
Product ids can be obtained by scanning your network with tinytuya if you do not have access to the iot.tuya.com portal to get the info from the cloud.